sdk/src/main/java/com/oursky/authgear/AuthenticateOptions.kt

```diff
@@ -10,18 +10,10 @@ data class AuthenticateOptions @JvmOverloads constructor(
      * Redirection URI to which the response will be sent after authorization.
      */
     var redirectUri: String,
-    /**
-     * OAuth 2.0 state value.
-     */
-    var state: String? = null,
```

Comment:
```
[I] $ rg --sort path 'var state: String\? = null'
sdk/src/main/java/com/oursky/authgear/AuthenticateOptions.kt
16:    var state: String? = null,

sdk/src/main/java/com/oursky/authgear/PromoteOptions.kt
11:    var state: String? = null,

sdk/src/main/java/com/oursky/authgear/ReauthenticateOptions.kt
16:    var state: String? = null,

sdk/src/main/java/com/oursky/authgear/app2app/App2AppAuthenticateOptions.kt
9:    var state: String? = null

sdk/src/main/java/com/oursky/authgear/data/HttpClient.kt
67:                    var state: String? = null

sdk/src/main/java/com/oursky/authgear/oauth/OidcAuthenticationRequest.kt
13:    var state: String? = null,
```

Please also remove `var state: String? = null` in ReauthenticateOptions and PromoteOptions

Reply:
removed `state` from ReauthenticateOptions and PromoteOptions